content/docs/dvclive/how-it-works.md

```diff
@@ -28,6 +28,7 @@ To illustrate with an example, given the following code:
 
 ```python
 import random
+from pathlib import Path
```

Comment:
No needed?
```suggestion
```

Reply:
I don't follow. The code snippet doesn't work without it, and other imports are included. Why include `import random` but not this?